<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrushEditor</class>
 <widget class="QWidget" name="BrushEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QComboBox" name="shapeSelect">
     <property name="toolTip">
      <string>Tip Shape</string>
     </property>
     <item>
      <property name="text">
       <string>Sphere</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cylinder</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cone</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bell</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mountains</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="LabelSlider" name="brushSizeLabel">
     <property name="text">
      <string>Size:</string>
     </property>
     <property name="buddy">
      <cstring>brushSize</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="brushSize">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="LabelSlider" name="strengthLabel">
     <property name="text">
      <string>Strength:</string>
     </property>
     <property name="buddy">
      <cstring>strength</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="strength">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="pressureMode">
     <property name="toolTip">
      <string>Pressure Mode</string>
     </property>
     <property name="currentText">
      <string>Air Brush</string>
     </property>
     <item>
      <property name="text">
       <string>Air Brush</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Constant</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Adjustable</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="LabelSlider" name="scaleLabel">
     <property name="text">
      <string>Scale:</string>
     </property>
     <property name="buddy">
      <cstring>strength</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="scaleField">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="randomizeButton">
     <property name="text">
      <string>Randomize</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>292</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelSlider</class>
   <extends>QLabel</extends>
   <header>labelslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
